src/containers/catalog/CatalogActionFactories.js

```diff
@@ -0,0 +1,23 @@
+/* @flow */
+import * as actionTypes from './CatalogActionTypes';
+import { EntitySet } from '../../components/entityset/EntitySetStorage';
```

Comment:
this shouldn't be necessary, but I'll have to double check on how Flow works again. we can revisit Flow at another time.